easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -56,6 +56,8 @@
 EASY_INSTALL_TARGET = "easy_install"
 EASY_INSTALL_INSTALL_CMD = "%(python)s setup.py " + EASY_INSTALL_TARGET + " --prefix=%(prefix)s %(installopts)s %(loc)s"
 PIP_INSTALL_CMD = "pip install --prefix=%(prefix)s %(installopts)s %(loc)s"
+if fancylogger.getLogger().level == fancylogger.logging.DEBUG:
```

Comment:
@smoors This makes sense, but you should use `if build_option('debug')` here rather than talking to `fancylogger`.

Also, you shouldn't do it here, but in `PythonPackage.__init__` where `self.install_cmd = PIP_INSTALL_CMD` is used.